TLM/TLM/Resources/Translations/Menu.csv

```diff
@@ -17,14 +17,14 @@
-"Roundabout.Option:No crossing inside","","Pedestrians shall not cross to the center of roundabout","","Pas de croisement à l'intérieur","","","","","","Pieszy nie powinni przechodzić przez drogę do środka ronda","","Запретить пешеходные переходы внутри круга","","","Pedestrians shall not cross to the center of roundabout","","Заборонити пішохідні переходи всередині"
-"Roundabout.Option:No crossing on incoming roads","","Pedestrians shall not cross the roads approaching the roundabout","","Pas de croisement sur les routes entrantes","","","","","","Pieszy nie powinni przechodzić przez drogę tuż przed rondem","","Запретить пешеходные переходы на подъездных дорогах к кругу","","","Pedestrians shall not cross the roads approaching the roundabout","","Заборонити пішохідні переході на під'їздних дорогах"
-"Roundabout.Option:Set priority signs","","Add priority signs on the roundabout junction","","Définir les panneaux de priorité","","","","","","Dodaj znaki pierwszeństwa przejazdu na rondzie","","Установить знаки приоритета","","","Add priority signs on the roundabout junction","","Встановити знаки пріоритету на круговому перехресті"
+"Tooltip.Keybinds:Change lane arrows","","Ctrl+Click: Separate turning lanes for entire junction \nAlt+Click: Separate turning lanes for segment","","Changer les flèches des voies","","","車線矢印を変更する","","","Ctrl+Klik: Oddziel pasy skrętu we wszystkich segmentach skrzyżowania \nAlt+Klik: Oddziel pasy skrętu wybranego segmentu","Ctrl+Clique: Separa todas as faixas de viragem para toda a bifurcação \nAlt+Clique: Separa as faixas de viragem por segmento","Ctrl+Клик: Отдельные полосы для поворотов на всём перекрёстке \nAlt+Клик: Отдельные полосы для поворотов на выбранном участке дороги","變更車道箭頭","Ctrl+点击: 整个路口的车道更改为单独行车导向箭头\nAlt+点击: 一段路口的车道更改为单独行车导向箭头","Ctrl+Click: Separate turning lanes for entire junction \nAlt+Click: Separate turning lanes for segment","Şerit oklarını değiştir","Ctrl+Клік: Роздільні стрілки на смугах руху для всіх доріг цього перехрестя \nAlt+Клік: Роздільні стрілки на смугах руху для сегменту"
+"Tooltip.Keybinds:Auto TL","","Ctrl+Click: Quick setup","","Feux automatiques","","","","","","Ctrl + Klik: Szybka instalacja","Ctrl+Clique: Configuração rápida","Автоматическое создание светофора с таймером","自動化號誌","Ctrl+点击: 快速设置","Ctrl+Click: Quick setup","Özdevinimli TI","Автоматичне створення світлофора з таймером"
+"Tooltip.Keybinds:Add priority signs","","Ctrl+Shift+Click: Quick setup","","Ajouter les panneaux de priorité","","","","","","Ctrl+Shift+Klik: Szybka instalacja","Ctrl+Shift+Clique: Configuração rápida","Добавить знаки приоритета на перекрёстках круга","加入優先標誌","Ctrl+Shift+点击: 快速设置","Ctrl+Shift+Click: Quick setup","Öncelik işaretleri ekle","Додати знаки приорітету руху"
```

Comment:
A problem right here: My own translations do not include shortcut keys.
We can either modify Rus and Ukr and include shortcuts, or modify the code (simple but incorrect) and make it build localized shortcut strings (correct but harder).

Reply:
Yeah, I saw that, mentioned about it before, but now I don't know what to do.

Reply:
Updated 3 strings above